include/swift/Threading/Impl/C11.h

```diff
@@ -95,9 +95,8 @@ struct lazy_mutex_handle {
   std::int32_t once; // -1 = initialized, 0 = uninitialized, 1 = initializing
 };
 
-inline constexpr lazy_mutex_handle lazy_mutex_initializer() {
-  return (lazy_mutex_handle){};
-}
+#define SWIFT_LAZY_MUTEX_INITIALIZER ((lazy_mutex_handle){})
```

Comment:
I think technically this is creating a temporary unnamed mutex value, which will then be copied into its destination. Obviously that will optimize out, but I wonder if this should be `{}` instead.

Reply:
I believe that this is the correct way to spell this in C.  The aggregate initializer will zero-initialize, but it needs the concrete type to compute the size.